scripts/lanenet_data_postprocess.py

```diff
@@ -0,0 +1,47 @@
+from absl import app, flags
```

Comment:
Can you please create a separate PR for this file because it is not related to this PR?